kong/plugins/datadog/handler.lua

```diff
@@ -2,143 +2,158 @@ local BasePlugin       = require "kong.plugins.base_plugin"
 local basic_serializer = require "kong.plugins.log-serializers.basic"
 local statsd_logger    = require "kong.plugins.datadog.statsd_logger"
 
-
 local ngx_log       = ngx.log
 local ngx_timer_at  = ngx.timer.at
 local string_gsub   = string.gsub
 local pairs         = pairs
 local string_format = string.format
 local NGX_ERR       = ngx.ERR
 
-
 local DatadogHandler    = BasePlugin:extend()
 DatadogHandler.PRIORITY = 10
 DatadogHandler.VERSION = "0.1.0"
 
-
-local get_consumer_id = {
-  consumer_id = function(consumer)
-    return consumer and string_gsub(consumer.id, "-", "_")
-  end,
-  custom_id   = function(consumer)
-    return consumer and consumer.custom_id
-  end,
-  username    = function(consumer)
-    return consumer and consumer.username
-  end
+local ConsumerId = {
+  consumer_id = function(consumer) return consumer and string_gsub(consumer.id, "-", "_") end,
+  custom_id   = function(consumer) return consumer and consumer.custom_id end,
+  username    = function(consumer) return consumer and consumer.username end
```

Comment:
I don't see any reason to make this change and we do not use camelcase for naming variables.

Reply:
I was following what appeared to be lua conventions above for `DatadogHandler` and `BasePlugin`. This table works as a module in this file so I wanted to use camelcase to demonstrate that.